lisa/features/infiniband.py

```diff
@@ -339,94 +346,97 @@ def install_ofed(self) -> None:
         kernel_version = node.tools[Uname].get_linux_information().kernel_version
         self._install_dependencies()
 
-        # Install OFED
-        ofed_version = self._get_ofed_version()
-        if isinstance(node.os, Oracle):
-            os_class = "ol"
-        elif isinstance(node.os, Redhat):
-            os_class = "rhel"
-        else:
-            os_class = node.os.name.lower()
+        if not isinstance(node.os, CBLMariner):
```

Comment:
Change to below, and add comments to explain why Mariner skipped.
```suggestion
        if isinstance(node.os, CBLMariner):
            return
```